opensrp-chw/build.gradle

```diff
@@ -51,6 +51,12 @@ android {
         buildConfigField "int", "MAX_SYNC_RETRIES", '3'
         buildConfigField "long", "BUILD_TIMESTAMP", System.currentTimeMillis() + "L"
         buildConfigField "boolean", "IS_SYNC_SETTINGS", "false"
+        buildConfigField "String", "THINKMD_BASE_URL", '"https://thinkmd-v4-app-b-ona.herokuapp.com/"'
+        buildConfigField "String", "THINKMD_END_POINT", '"https://fhir-v4-staging-thinkmd-tech.herokuapp.com/fhir"'
+        buildConfigField "String", "PATIENT_IDENTIFIER", '"https://fhir.smartregister.org/thinkMD_id"'
```

Comment:
I think will not be changing per thinkmd integration and should be defined as constants on `ThinkMDConfig`